x-pack/test_serverless/functional/test_suites/security/cypress/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "author": "Elastic",
+  "name": "@kbn/security-solution-plugin",
```

Comment:
This name looks wrong to me because it's already used by the `security_solution` plugin.

Reply:
@YulNaumenko which name should be used here? Thanks!